.github/workflows/ci.yml

```diff
@@ -363,6 +363,10 @@ jobs:
           package_json_path="$PLUGIN_DIRECTORY/package.json"
           version=$(jq -r .version "$package_json_path")
           pr_version="$version+${PLUGIN_VERSION_SUFFIX}"
+
+          # Version in the DB is at most 190 characters
+          pr_version="${pr_version:0:190}"
```

Comment:
@xnyo Would there be a case when user creates too long version and we overwrite it in the package json and drift happens between plugin.json and package.json? Highly unlikely?

Reply:
we should fail CI in this case. this is almost like a silent fail